subxt/src/lib.rs

```diff
@@ -33,23 +33,31 @@
 )]
 #![allow(clippy::type_complexity)]
 
+#[cfg(any(
+    all(feature = "web", feature = "native"),
```

Comment:
> subxt: exactly one of the 'web' and 'native' features should be used.

should this be `not(all(feature = "web", feature = "native"))` then?

Reply:
Hmmm, I think:

`not(all(feature = "web", feature = "native"))` specifically matches when there are not exactly both features enabled. Ie if 1 feature is enabled, it would be true.

`not(any(feature = "web", feature = "native"))` matches when there are not any of the features enabled, ie none of them are activated.

Reply:
I mean that we would need something similar to:

```rust
#[cfg(all(
    // ensure that only of one `web` and `native` is used
    not(all(feature = "web", feature = "native")),
    // ensure that at least one of the feature `web` or `native` is enabled.
    any(feature = "web", feature = "native")
))]
```